tools/osv-linter/internal/linter.go

```diff
@@ -95,6 +94,10 @@ func LintCommand(cCtx *cli.Context) error {
 	// Figure out what files to check.
 	var filesToCheck []string
 	for _, thingToCheck := range cCtx.Args().Slice() {
+		// Treat "-" as stdin.
+		if thingToCheck == "-" {
+			thingToCheck = "/dev/stdin"
```

Comment:
I assume this won't work on Windows. Can we use the more platform agnostic `os.Stdin` here instead? We can't add it to `filesToCheck` though, because `os.Stdin` represents an open file.

Reply:
Good callout, thanks. I think ae463a5 strikes the right balance, UX-wise.